cmd/autoheal/healer.go

```diff
@@ -147,8 +147,17 @@ func (b *HealerBuilder) Build() (h *Healer, err error) {
 	h.rulesQueue = workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "rules")
 	h.alertsQueue = workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "alerts")
 
-	// initialize active jobs map:
-	h.activeJobs = new(syncmap.Map)
+	h.jobRunners = map[JobRunnerType]JobRunner{
+		AWX: AWXJobRunner{
+			config: h.config.AWX(),
```

Comment:
When constructing the implementations It would be clean if they each get the entire configuration, and select the pieces relevant to them.

Reply:
Nice thanks @moolitayer